advancemenu.sh

```diff
@@ -134,227 +131,212 @@ function valheim_server_install() {
     done
-    echo ""
-    # Take user input for Sow Server Public
-    echo ""
-    tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-    tput setaf 2; echo "---------------------Public Server Shown--------------------" ; tput setaf 9;
-    tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-    tput setaf 1;  echo "Do you want your Server Shown Publicly" ; tput setaf 9;
-    tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-    tput setaf 2;  echo "Show Server in Public List: 1 " ; tput setaf 9;
-    tput setaf 1;  echo "Play Server on LAN or Not Listed Publicly: 0" ; tput setaf 9;
-    tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-    echo ""
-      read -p "Set Public to 1 or 0: " publicList
-    tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-    echo ""
-    clear
-cat >> /home/steam/serverSetup.txt <<EOF
-Here is the information you entered
-This information is for you to ref later, in case you forgot
----------------------------------------------------------------
-nonroot steam password:  $userpassword
-Public Server Name:      $displayname
-Local World Name:        $worldname
-Valheim Server Password: $password
-Show Public: $publicList
----------------------------------------------------------------
-Each time this is ran, the past info will be added to each line
----------------------------------------------------------------
-EOF
-chown steam:steam /home/steam/serverSetup.txt
-clear
-echo "Here is the information you entered"
-echo "This information is saved in the valheim_server.sh file"
-echo "This information is saved in /home/steam/serverSetup.txt for referance later, if you forget"
-tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
-tput setaf 2; echo "nonroot steam password:  $userpassword " ; tput setaf 9;
-tput setaf 2; echo "Public Server Name:      $displayname " ; tput setaf 9;
-tput setaf 2; echo "Local World Name:        $worldname " ; tput setaf 9;
-tput setaf 2; echo "Valheim Server Password: $password " ; tput setaf 9; 
-tput setaf 2; echo "Show Public Server: $publicList " ; tput setaf 9; 
-tput setaf 2; echo "------------------------------------------------------------" ; tput setaf 9;
+        # Take user input to Show Server Public
+    echo ""
+    tput setaf 2; echo "$DRAW60" ; tput setaf 9;
+    tput setaf 2; echo "$PUBLIC_ENABLED_DISABLE_HEADER" ; tput setaf 9;
+    tput setaf 2; echo "$DRAW60" ; tput setaf 9;
+    tput setaf 1; echo "$PUBLIC_ENABLED_DISABLE_INFO" ; tput setaf 9;
+    tput setaf 2; echo "$DRAW60" ; tput setaf 9;
+    tput setaf 2; echo "$PUBLIC_ENABLED_DISABLE_EXAMPLE_SHOW" ; tput setaf 9;
+    tput setaf 1; echo "$PUBLIC_ENABLED_DISABLE_EXAMPLS_LAN_NO_SHOW" ; tput setaf 9;
```

Comment:
Typo fix available in https://github.com/Nimdy/Dedicated_Valheim_Server_Script/pull/216.